src/keylessBackup/KeylessBackupProgress.tsx

```diff
@@ -285,80 +307,150 @@ function Setup({ origin }: { origin: KeylessBackupOrigin }) {
 
   const onPressManualOnboarding = () => {
     ValoraAnalytics.track(KeylessBackupEvents.cab_progress_failed_manual, { origin })
-    navigate(Screens.AccountKeyEducation)
+    isOnboarding
+      ? goToNextOnboardingScreen({
+          firstScreenInCurrentStep: Screens.SignInWithEmail,
+          onboardingProps: { ...onboardingProps, showRecoveryPhraseEducation: true },
+        })
+      : navigate(Screens.AccountKeyEducation)
   }
 
   const onPressSkip = () => {
     ValoraAnalytics.track(KeylessBackupEvents.cab_progress_failed_skip_onboarding)
-    navigate(Screens.ChooseYourAdventure)
+    isOnboarding
```

Comment:
We should only see the skip button in onboarding, so shouldn't need this check.

Reply:
Fixed in 03ba34c68ada224b2e007793d3f03ace442c3a25!